apps/web/src/components/Feed/FeedEventItem.tsx

```diff
@@ -0,0 +1,144 @@
+import { useCallback } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import breakpoints from '~/components/core/breakpoints';
+import { VStack } from '~/components/core/Spacer/Stack';
+import { FEED_EVENT_ROW_WIDTH_DESKTOP } from '~/components/Feed/dimensions';
+import { FeedEventData } from '~/components/Feed/FeedEventData';
+import { FeedEventSocializeSection } from '~/components/Feed/Socialize/FeedEventSocializeSection';
+import { FeedMode } from '~/components/Feed/types';
+import { FeedEventItemFragment$key } from '~/generated/FeedEventItemFragment.graphql';
+import { FeedEventItemQueryFragment$key } from '~/generated/FeedEventItemQueryFragment.graphql';
+import { FeedEventItemWithErrorBoundaryFragment$key } from '~/generated/FeedEventItemWithErrorBoundaryFragment.graphql';
+import { FeedEventItemWithErrorBoundaryQueryFragment$key } from '~/generated/FeedEventItemWithErrorBoundaryQueryFragment.graphql';
+import { ReportingErrorBoundary } from '~/shared/errors/ReportingErrorBoundary';
+import { TriedToRenderUnsupportedFeedEvent } from '~/shared/errors/TriedToRenderUnsupportedFeedEvent';
+
+type FeedEventItemProps = {
+  eventRef: FeedEventItemFragment$key;
+  queryRef: FeedEventItemQueryFragment$key;
+  feedMode: FeedMode;
+};
+
+function FeedEventItem({ eventRef, queryRef, feedMode }: FeedEventItemProps) {
```

Comment:
This used to be `FeedItem`, I broke it apart to `FeedEventItem` and `PostItem` since FeedEvent and Post rendering logic are starting to diverge